src/service/owners-impl.js

```diff
@@ -15,20 +15,48 @@
  */
 
 /* eslint-disable no-unused-vars */
+import {Resource, ResourceState} from './resource';
+import {Services} from '../services';
+import {dev, devAssert} from '../log';
+import {isArray} from '../types';
+import {registerServiceBuilderForDoc} from '../service';
+
+const TAG_ = 'Owners';
```

Comment:
nit: TAG

I know we use them interchangeably, but TAG without underscore is dominant.